tests/neptune/internal/storage/test_datastream.py

```diff
@@ -63,7 +63,9 @@ def test_generate_chunks_from_stream(self):
         text = u"ABCDEFGHIJKLMNOPRSTUWXYZ"
 
         # when
-        stream = FileChunkStream(UploadEntry(StringIO(text), "some/path"))
+        stream = FileChunkStream(
+            UploadEntry(BytesIO(bytes(text, "utf-8")), "some/path")
```

Comment:
this looks like a part of a different PR ;)

Reply:
that is true.Will move it